app/graphql/types/comment_type.rb

```diff
@@ -7,5 +7,6 @@ class CommentType < GraphQL::Schema::Object
     field :article_id, ID, null: false
     field :created_at, String, null: false
     field :image, String, null: false
+    field :article, ArticleType, null: false
```

Comment:
Shouldn't need this field, nor article_id as you are creating a comment via the article slug

Reply:
I have removed article_id with no problems but if I remove article my queries still work but my test is failing.